pennylane/ops/qutrit/matrix_ops.py

```diff
@@ -91,6 +91,8 @@ def compute_matrix(U):  # pylint: disable=arguments-differ
         The canonical matrix is the textbook matrix representation that does not consider wires.
         Implicitly, this assumes that the wires of the operator correspond to the global wire order.
 
+        .. seealso:: :meth:`~.QutritUnitary.matrix`
```

Comment:
It's added to be inline with the documentation for `compute_matrix()` from other operations. I forgot to add it during the PR that added `QutritUnitary`, so I added it here because it was a minor change.